bids2openminds/converter.py

```diff
@@ -32,8 +33,42 @@ def convert(input_path,  save_output=False, output_path=None, multiple_files=Fal
 
     dataset_description = utility.read_json(dataset_description_path.iat[0, 0])
 
+    if short_name is None:
+        while True:
+            input_name = input(
+                "To convert this dataset, a short name is required.\n"
+                "Please enter an informative short name (less than 10 characters).\n"
+                "If you don't want to assign one now, press Enter, "
+                "lookup labels and internal identifiers will be the same.\n> "
+            ).strip()
+
+            # User pressed Enter → skip assigning
+            if not input_name:
+                short_name = None
+                break
+
+            # Check length
+            if len(input_name) < 10:
+                short_name = input_name
+                break
+            else:
+                print(
+                    "The short name must be fewer than 10 characters. Please try again.\n")
+
+    elif short_name is False:
```

Comment:
Can't be none later needs to be defined (is a required property and needs to be set as a default)